maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -2534,10 +2544,10 @@ private TestClassPath generateTestClasspath() {
     }
 
     /**
-     * Return a new set containing only the artifacts accepted by the given filter.
+     * Return a new set containing only the artifacts not accepted by the given filter.
      *
      * @param artifacts The unfiltered artifacts
-     * @param filter    The filter to apply
+     * @param filter    The excludes filter to apply
      * @return The filtered result
      */
     private static Set<Artifact> filterArtifacts(Set<Artifact> artifacts, ArtifactFilter filter) {
```

Comment:
I think the second param needs to be renamed.

Reply:
Can be further improved, but not really related or blocking I think. Important is that the description is correct now.